include/convex_bodies/hpolytope.h

```diff
@@ -1021,15 +1022,39 @@ class HPolytope {
             }
     }
 
-    template <typename update_parameters>
-    NT compute_reflection(Point &v, const Point &, DenseMT const &AE, VT const &AEA, NT const &vEv, update_parameters const &params) const {
+    // function to compute reflection for GaBW random walk
+    // compatible when the polytope is both dense or sparse
+    template <typename AE_type, typename update_parameters>
+    NT compute_reflection(Point &v, Point &p, AE_type const &AE, VT const &AEA, NT &vEv, update_parameters const &params) const {
```

Comment:
What is `AE_type` ? I guess it is a template that can be instantiated as a Dense or Sparse eigen matrix type, right? If this is the case why not naming in a way that specifies its use, like `DenseSparseMT` or something like that?

Reply:
sure thing, will do!